Add note about HMR WebSocket with express to upgrading (#30905)
vercel/next.js: docs/upgrading.md

}
```

> For custom servers like express you might need to use `app.all` to ensure the request is passed correctly, for example:

Reviewer: ```suggestion
For custom servers, such as `express`, you may need to use `app.all` to ensure the request is passed correctly, for example:
```
